client/my-sites/activity/activity-log/threat-alert.jsx

```diff
@@ -28,6 +32,10 @@ import './threat-alert.scss';
 const debug = debugFactory( 'calypso:activity-log' );
 
 const detailType = threat => {
+	if ( threat.hasOwnProperty( 'table' ) ) {
```

Comment:
I think the table stuff should move out into it's own PR as part of that work? Probably just here still from the meetup work I'm guessing.

Reply:
Yes, this remained in my codebase from the meetup work. I'll take it out of this.